src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/common/util/RandomDataGeneratorsTest.kt

```diff
@@ -7,22 +7,22 @@ private const val TEST_REPEATS = 20
 
 class RandomDataGeneratorsTest {
   @RepeatedTest(TEST_REPEATS)
-  fun `randomPrisonNumber should create random prison numbers`() {
-    randomPrisonNumber() shouldMatch "^[A-Z][0-9]{4}[A-Z]{2}$"
+  fun `should create random prison numbers`() {
+    prisonNo() shouldMatch "^[A-Z][0-9]{4}[A-Z]{2}$"
```

Comment:
Can we call this `prisonNumber` please? There's been a bit of confusion with `prisonNumbers`s, `prisonId`s and `prisonerNumbers` and I think it would help if we spelled this out.

Reply:
It'd break the factory-generation on the `prisonNumber` variable if we renamed it to this. However, given that the generation strategy is random for each of these data-generation methods, it makes more sense to stick the `random` prefix back in and rename the file itself to just `DataGenerators.kt`. Given that we statically import those methods it makes more sense to prefix them with `random` anyway.